packer/ubuntu-kvm-amd64host.json

```diff
@@ -0,0 +1,95 @@
+{
+  "min_packer_version": "1.6.0",
+  "variables": {
+    "vm_name": "JMU Linux Mint",
+    "semester": "Fa20",
+    "version": "20",
+
+    "build_id": "{{isotime \"2006-01-02\"}}",
+
+    "git_repo": "https://github.com/jmunixusers/cs-vm-build",
+    "git_branch": "main",
+
+    "headless": "false",
+    "audio": "pulse",
+
+    "output_dir": "{{pwd}}/artifacts_mint",
+    "mirror_url": "http://cdimage.ubuntu.com/focal/daily-live/current",
+    "iso_file": "focal-desktop-arm64.iso",
+
+    "ssh_user": "oem",
+    "ssh_pass": "oem"
+  },
+  "builders": [
+    {
+      "type": "qemu",
+      "headless": "{{user `headless`}}",
+      "cpus": 4,
+      "memory": 4096,
+      "firmware": "/usr/share/qemu-efi-aarch64/QEMU_EFI.fd",
+      "qemu_binary": "qemu-system-aarch64",
+      "qemuargs": [
+        [ "-boot", "strict=off" ],
+        [ "-machine", "virt" ],
+        [ "--accel", "tcg,thread=multi" ],
+        [ "-vga", "vmware" ],
+        [ "-cpu", "cortex-a57" ],
+        [ "-display", "gtk" ],
```

Comment:
FWIW, this seems to cause issues with building on macOS. `cocoa` is probably the right value there. But there also seem to be issues with the `--accel` flag on macOS (because it probably needs to be `hvf` but as of 6.0 `qemu` doesn't support `hvf` on Apple Silicon without patches)

Reply:
I'll probably delete this file in the final PR. Emulation has never been fast enough to be useful here.